webapp/package.json

```diff
@@ -28,7 +28,7 @@
     "@angular/platform-browser-dynamic": "18.2.1",
     "@angular/router": "18.2.1",
     "@devolutions/icons": "4.0.8",
-    "@devolutions/iron-remote-desktop": "^0.2.0",
+    "@devolutions/iron-remote-desktop": "file:~/Projects/ironrdp/web-client/iron-remote-desktop/dist",
```

Comment:
Waiting for https://github.com/Devolutions/IronRDP/pull/818 to be merged and released